README.md

```diff
@@ -1 +1,170 @@
-# Leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖Leaderboard <a name="about-project"></a>
+
+
+
+**Leaderboard** is a wonderful  project to see the score of the users.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>HTML ,CSS and JavaScript</summary>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+
+-  **input fields**
+- **button**
+- **dynamic list**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] [OPITONAL] As a developer, it's important to not only write clean and well-organized code but also to document your work effectively. One key aspect of this is creating a detailed and informative README file. In particular, the Key Features & Future Features section of your README presents a great opportunity to showcase your creativity and vision for the project. I would highly recommend adding a compelling and descriptive feature to this section to grab the attention of recruiters or future clients. 😊